launch.sh

```diff
@@ -9,38 +9,32 @@ source "$(dirname "$0")/ansible-env.bash"
 
 CEPH_ANSIBLE=~/ceph-ansible/
 NUKE=0
+DESTROY=0
 LOG=OUTPUT
 YML="$(dirname "$0")/linode.yml"
 RETRY="${YML%.*}.retry"
 
 function main {
-    if [ "$NUKE" -gt 0 ]; then
+    if [ "$DESTROY" -gt 0 ]; then
+        time python2 "$(dirname "$0")/linode-destroy.py"
+    elif [ "$NUKE" -gt 0 -o ! -f ansible_inventory ]; then
         time python2 "$(dirname "$0")/linode-nuke.py"
     fi
-    if [ "$NUKE" -gt 0 -o ! -f ansible_inventory ]; then
+    if [ "$NUKE" -gt 0 -o "$DESTROY" -gt 0 -o ! -f ansible_inventory ]; then
         time python2 "$(dirname "$0")/linode-launch.py"
     fi
     # wait for Linodes to finish booting
     time python2 "$(dirname "$0")/linode-wait.py"
+    sleep 5
```

Comment:
This replaces a loop to run ansible playbooks over and over again.  Often this just delays finding out what's wrong, because the same error keeps happening.  linode-wait.py pretty much ensures that the linodes are up, the 5 seconds is just to give it a little extra time to finish booting.

Reply:
Maybe add a loop to `linode-wait` to ssh to each node and just do "echo hello world".